examples/src/demos/index.ts

```diff
@@ -0,0 +1,6 @@
+import { lazy } from 'react'
+
+export default {
+  TakeControl: { descr: '', tags: [], Component: lazy(async () => await import('./TakeControl')), bright: false },
+  Bubbles: { descr: '', tags: [], Component: lazy(async () => await import('./Bubbles')), bright: false },
+}
```

Comment:
Demos are defined here and will import the `index.tsx` within the specified folder due to normal module resolution. It is better to have a folder per Demo though so that they can be a bit more expansive.